src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.softserve.actent.service.impl;
+
+import com.softserve.actent.model.entity.Review;
+import com.softserve.actent.repository.ReviewRepository;
+import com.softserve.actent.service.ReviewService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewServiceImpl implements ReviewService {
+
+    @Autowired
+    ModelMapper modelMapper;
+
+    @Autowired
+    ReviewRepository reviewRepository;
+
+    @Override
+    public Review addReview(Review review) {
+
+        return reviewRepository.save(review);
+    }
+
+    @Override
+    public Review getReviewById(Long reviewId) {
+
+        if (reviewRepository.findById(reviewId).isPresent()) {
+            return reviewRepository.findById(reviewId).get();
```

Comment:
"reviewRepository.findById(reviewId)" calling DB 2 times (line28,29) for the same purpose

Reply:
Fixed, thank you. 👍